src/app/clusters/thermostat-server/thermostat-delegate.h

```diff
@@ -204,7 +186,7 @@ class Delegate
      * @return CHIP_ERROR if there was an error adding the thermostat suggestion to the list.
      */
     virtual CHIP_ERROR
-    AppendToThermostatSuggestionsList(const ThermostatSuggestionStructWithOwnedMembers & thermostatSuggestion) = 0;
+    AppendToThermostatSuggestionsList(const Structs::ThermostatSuggestionStruct::Type & thermostatSuggestion) = 0;
```

Comment:
Should document, for this and remove, that the caller will handle dirty-marking.

Reply:
done.